pkgs/by-name/md/mdbook-pandoc/package.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, fetchpatch
+, stdenv
+, rustPlatform
+, pandoc
+, texliveSmall
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mdbook-pandoc";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "max-heller";
+    repo = pname;
```

```suggestion
    repo = "mdbook-pandoc";
```
`repo = pname` is discoraged #277994